libs/community/langchain_community/agent_toolkits/financial_datasets/toolkit.py

```diff
@@ -3,9 +3,9 @@
 from typing import List
 
 from langchain_core.pydantic_v1 import Field
-from langchain_core.tools import BaseToolkit
+from langchain_core.tools import BaseTool
+from langchain_core.tools.base import BaseToolkit
```

Comment:
Why it is not
`from langchain_core.tools import BaseToolkit`
to make a paritet with 
`from langchain_core.tools import BaseTool`
?